helpers/handlebars/helpers/index.js

```diff
@@ -33,6 +33,13 @@ module.exports = {
             return opts.inverse(this);
         }
     },
+    ifvalue: (conditional, options) => {
```

Comment:
Nein, der Unterschied in den Helpern ist == bei ifeq & === bei ifvalue.
Jedoch sollte man ifvalue eventuell mal auf ifeq anpassen, damit die syntaxa gleich ist